sphinx/templates/latex/latex.tex_t

```diff
@@ -30,6 +30,8 @@
 <%= contentsname %>
 <%= numfig_format %>
 <%= pageautorefname %>
+\newcommand{\sphinxVerbatimContinuedText}{\footnotesize(<%= _('continued from previous page') %>)}
+\newcommand{\sphinxVerbatimContinuesText}{\footnotesize(<%= _('continues on next page') %>)}
```

Comment:
I think .sty file is usually used for modify styles and macros in LaTeX world.
So I feel `preamble` key is better to do that.

Note: it might be better to allow user's .sty file instead of `preamble` key.
Now the key became much important, but it is boring to maintain it in conf.py.